docs/concepts/editions-overview.md

```diff
@@ -6,11 +6,11 @@
 
 .NET Core is a cloud-optimized, cross-platform implementation of the .NET Platform. It currently supports three main operating systems: Linux, Windows and OS X.
 
-There are several characteristics of .NET Core:
+The following are main characteristics of .NET Core:
```

Comment:
There are some, but I don't want to actually add them here. Trying for the "impassionate overview" style & mood. :)